src/net/l4/tcp.c

```diff
@@ -438,7 +415,7 @@ static void send_rst_reply(struct sk_buff *skb) {
  */
 static void send_nonseq_from_sock(struct tcp_sock *tcp_sk,
 		struct sk_buff *skb) {
-	debug_print(9, "send_nonseq_from_sock: send %p\n", skb);
+	log_debug("send_nonseq_from_sock: send %p", skb);
```

Comment:
There is no need to print to print function name by hand since this job is already done by log_debug. The same is for other log_debug's below.